apps/web/src/pages/projects/[project]/export.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { PageLayout } from '../../../components/ui/page-layout';
+
+import { Button } from '../../../components/ui/button';
+import { Heading } from '@/components/ui/typography';
+import { Separator } from '@/components/ui/separator';
+import { useRouter } from 'next/router';
+import { useProject } from '@/hooks/use-project';
+import useVotes from '@/hooks/use-votes';
+import useIdeas from '@/hooks/use-ideas';
+import useTags from '@/hooks/use-tags';
+import useComments from '@/hooks/use-comments';
+import usePolls from '@/hooks/use-poll';
+
+export default function ProjectExport() {
+  const router = useRouter();
+  const { project } = router.query;    
+
+  // Function to export data as a file
+  const exportData = (data: any, fileName: any, type: any) => {
```

Comment:
Kunnen we de types van data, filename en type niet vast stellen?

Bijvoorbeeld data:<EenJsonObjectType met minimaal x velden>, fileName: string, type: string